Source/WebKit/UIProcess/win/PageClientImpl.cpp

```diff
@@ -51,7 +51,9 @@ PageClientImpl::PageClientImpl(WebView& view)
 std::unique_ptr<DrawingAreaProxy> PageClientImpl::createDrawingAreaProxy(WebProcessProxy&)
 {
 #if USE(GRAPHICS_LAYER_WC)
-    if (m_view.page()->preferences().useGPUProcessForWebGLEnabled())
+    if (m_view.page()->preferences().useGPUProcessForDOMRenderingEnabled()
+        || m_view.page()->preferences().useGPUProcessForCanvasRenderingEnabled()
+        || m_view.page()->preferences().useGPUProcessForWebGLEnabled())
```

Comment:
Revert this part. If WebGL is running in web process, the compositor can't get the output texture. WebGL and the compositor should be in the same process for WinCairo.